src/js/components/ServiceSearchFilter.js

```diff
@@ -0,0 +1,60 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
+
+import FilterInputText from './FilterInputText';
+import QueryParamsMixin from '../mixins/QueryParamsMixin';
+import ServiceFilterTypes from '../constants/ServiceFilterTypes';
+
+const METHODS_TO_BIND = ['setSearchString'];
+
+class ServiceSearchFilter extends mixin(QueryParamsMixin) {
+  constructor() {
+    super();
+
+    this.state = {
+      searchString: ''
+    };
+
+    METHODS_TO_BIND.forEach(method => {
```

Comment:
Please don't omit the parentheses, even if only one parameter is present to stay consistent with the rest of our code base.

**Examples**
![arrow-functions](https://cloud.githubusercontent.com/assets/647035/14879706/1531031e-0d2b-11e6-97f1-a3ed25c6af21.png)